.github/workflows/main.yml

```diff
@@ -0,0 +1,35 @@
+name: automatic-test-coverage-report
```

Comment:
This workflow should simply be named "run unit tests" or something. Reporting coverage is a part of running the test suite.